pkg/apis/autoscaling/v1alpha1/pa_lifecycle.go

```diff
@@ -179,6 +191,18 @@ func (pas *PodAutoscalerStatus) MarkResourceFailedCreation(kind, name string) {
 		fmt.Sprintf("Failed to create %s %q.", kind, name))
 }
 
+// MarkContainerExiting changes the "ContainerHealthy" condition to false and records the exit code.
+func (pas *PodAutoscalerStatus) MarkContainerExiting(exitCode int32, message string) {
+	exitCodeString := fmt.Sprintf("ExitCode%d", exitCode)
+	message = fmt.Sprintf("Container failed with: %s", message)
```

Comment:
just `"Container failed with: " + message` -- no reason to waste cycles on formatting.

Reply:
How much difference is it?  I don't want to make the style inconsistent just for a few cycles.